### PR TITLE
Makefile: Fix osbuild-image-tests build

### PR DESCRIPTION
**Pull request by bcl**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ build:
 	go test -c -tags=integration -o osbuild-weldr-tests ./internal/weldrcheck/
 	go test -c -tags=integration -o osbuild-dnf-json-tests ./cmd/osbuild-dnf-json-tests/main_test.go
 	go test -c -tags=integration -o osbuild-rcm-tests ./cmd/osbuild-rcm-tests/main_test.go
-	go test -c -tags=integration -o osbuild-image-tests ./cmd/osbuild-image-tests/main_test.go
+	go test -c -tags=integration,travis -o osbuild-image-tests ./cmd/osbuild-image-tests/
```
Comment (larskarlitski) [inline Makefile]: I don't understand. I thought the point of the `travis` tag is to have that only set on travis?

**Comment by ondrejbudai** [comment]
I don't know about this, my plan was to remove the travis tag as soon as we remove Travis CI. However, I can imagine that it's convenient to run the tests directly from the source... Is that your use case, @bcl ?

File: Makefile (hunk 1 above)
Comment (bcl) [reply Makefile:14]: Building without travis results in:

```
go test -c -tags=integration -o osbuild-image-tests ./cmd/osbuild-image-tests/main_test.go
# command-line-arguments [command-line-arguments.test]
cmd/osbuild-image-tests/main_test.go:42:9: undefined: getOsbuildCommand
cmd/osbuild-image-tests/main_test.go:99:22: undefined: imageInfoPath
cmd/osbuild-image-tests/main_test.go:126:20: undefined: netNS
cmd/osbuild-image-tests/main_test.go:133:9: undefined: privateKeyPath
cmd/osbuild-image-tests/main_test.go:172:31: undefined: netNS
cmd/osbuild-image-tests/main_test.go:198:9: undefined: withNetworkNamespace
cmd/osbuild-image-tests/main_test.go:198:38: undefined: netNS
cmd/osbuild-image-tests/main_test.go:203:11: undefined: withBootedQemuImage
cmd/osbuild-image-tests/main_test.go:208:11: undefined: withBootedNspawnImage
cmd/osbuild-image-tests/main_test.go:213:11: undefined: withExtractedTarArchive
cmd/osbuild-image-tests/main_test.go:213:11: too many errors
make: *** [Makefile:120: build] Error 2
```

(also, this appears to have already been merged)

**Comment by ondrejbudai** [comment]
Yeah, this commit got merged with PR #367, closing this.

**Comment by teg** [comment]
Ah, I seem to have missed that patch in #367. Seems something is still not right, either the tag should be renamed or it should not be used from the make file... (really sounds like this should only be needed on travis).

**Comment by ondrejbudai** [comment]
Weird, I tried to reproduce without the travis tag and it worked. I don't personally use `Makefile`, so I don't mind the travis tag there.

**Pull request closed without merging**